bin/stacks/routing-dashboard-stack.ts

```diff
@@ -401,30 +400,6 @@ export class RoutingDashboardStack extends cdk.NestedStack {
               stat: 'Average',
             },
           },
-          {
-            type: 'metric',
-            x: 0,
-            y: 66,
-            width: 24,
-            height: 9,
-            properties: {
-              view: 'timeSeries',
-              stacked: false,
-              metrics: [
-                ['AWS/Lambda', 'Errors', 'FunctionName', poolCacheLambdaName],
-                ['.', 'Invocations', '.', '.'],
-                ...(ipfsPoolCacheLambdaName
-                  ? [
-                      ['AWS/Lambda', 'Errors', 'FunctionName', ipfsPoolCacheLambdaName],
-                      ['.', 'Invocations', '.', '.'],
-                    ]
-                  : []),
-              ],
-              region: region,
-              title: 'Pool Cache Lambda Error/Invocations | 5min',
-              stat: 'Sum',
-            },
-          },
```

Comment:
Why remove the dashboard entirely? We could just have the same dashboard, but with lines for each chain

Reply:
I'll add back